src/data/models/institure_holidays.js

```diff
@@ -4,15 +4,17 @@ import DataTypes from 'sequelize';
 import Model from '../sequelize';
 
 const Institure_Holidays = Model.define('institure_holidays', {
-  id: {
-    type: DataTypes.INTEGER(11).UNSIGNED,
-    allowNull: false,
-    autoIncrement: true,
-    primaryKey: true,
-  },
+  // required?
```

Comment:
The problem with sequelize data models is they expect a column with primary key, typically a column named `id`. 
If there exists no column which is primary key, sequelize created its own column named `id`. ( [Source](https://stackoverflow.com/a/29234106) )